### PR TITLE
adding attribute for ELB resource

### PR DESCRIPTION
**Pull request by Rohit1509**
Signed-off-by: Rohit Joshi <rohit.prasad.joshi@sap.com>

### Description

Please describe what this change achieves. Ensure you have read the [Contributing to InSpec AWS](https://github.com/inspec/inspec-aws/CONTRIBUTING.md) document before submitting.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [ ] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [x] Documentation provided or updated for resources 
- [x] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by sathish-progress — APPROVED** [review]
LGTM

File: docs/resources/aws_elb.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ See also the [AWS documentation on Elastic Load Balancing](https://docs.aws.amaz
 |protocols                              | A list of the protocols configured for the listeners of the load balancer. |
 |cross\_zone\_load\_balancing\_enabled? | The cross-zone load balancing status for ELB. |
 |access\_log\_enabled?                  | The access log status for ELB. |
+|certificate\_id                        | A list of SSL certificate ids configured for the listeners of the load balancer. |
```
Comment (IanMadd) [inline docs/resources/aws_elb.md:49]:
```suggestion
|certificate\_id                        | A list of SSL certificate IDs configured for the listeners of the load balancer. |
```

Comment (Rohit1509) [reply docs/resources/aws_elb.md]: fixed

Comment (Rohit1509) [reply docs/resources/aws_elb.md]: @IanMadd

**Pull request merged**